ocaml/sdk-gen/go/gen_go_binding.ml

```diff
@@ -0,0 +1,72 @@
+(* Copyright (c) Cloud Software Group, Inc.
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU Lesser General Public License as published
+   by the Free Software Foundation; version 2.1 only. with the special
+   exception on linking described in file LICENSE.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Lesser General Public License for more details.
+*)
+
+open CommonFunctions
+open Gen_go_helper
+
+let render_enums enums destdir =
+  let header =
+    render_template "FileHeader.mustache" (`O [("modules", `Null)])
+  in
+  let enums = render_template "Enum.mustache" enums |> String.trim in
+  let rendered = header ^ "\n" ^ enums ^ "\n" in
+  generate_file rendered destdir "enums.go"
+
+let render_api_messages_and_errors destdir =
+  let obj =
+    `O
+      [
+        ("api_errors", `A Json.api_errors)
+      ; ("api_messages", `A Json.api_messages)
+      ; ("modules", `Null)
+      ]
+  in
+  let header = render_template "FileHeader.mustache" obj ^ "\n" in
+  let error_rendered =
+    header ^ render_template "APIErrors.mustache" obj ^ "\n"
```

Comment:
Just an idea: you could use `String.concat`. `render_template` could take an optional parameter `?(nl=false)` that adds a newline at the end of the rendered string as it's a common pattern to add it.

Reply:
Yeah, it's a nice way.